pkg/devfile/adapters/kubernetes/utils/utils.go

```diff
@@ -156,10 +156,19 @@ func UpdateContainersWithSupervisord(devfileObj devfileParser.DevfileObj, contai
 			// This is done, so supervisord can use it in it's program
 			if !isEnvPresent(container.Env, adaptersCommon.EnvOdoCommandRun) {
 				klog.V(4).Infof("Updating container %v env with run command", container.Name)
+				var setEnvVariable, command string
```

Comment:
Please update the unit test for this function.

Reply:
added unit test for env variables.